lib/screens/home/proposals/proposals_details/address_card.dart

```diff
@@ -2,7 +2,7 @@ import 'package:provenance_wallet/common/pw_design.dart';
 import 'package:provenance_wallet/screens/home/proposals/proposals_screen/proposals_bloc.dart';
 import 'package:provenance_wallet/util/address_util.dart';
 import 'package:provenance_wallet/util/get.dart';
-import 'package:url_launcher/url_launcher.dart';
+import 'package:url_launcher/url_launcher_string.dart';
```

Comment:
What's the difference between the two?

Reply:
...gonna have to fix my newest pull request.

Reply:
the old one is deprecated